### ISSUE TITLE
toolbarSticky does not appear to be working in bootstrap modal

### ISSUE THREAD

**Issue by lehti29** [froala/wysiwyg-editor#4084]
##### Expected behavior.
Toolbar should be fixed to the top of viewport when scrolling, when froala editor exists in a bootstrap modal if toolbarSticky is set to true.

##### Actual behavior.
Toolbar is not fixed to the top of viewport when in bootstrap modal. 

##### Steps to reproduce the problem.
1. Initialize a froala editor instance within a bootstrap (v3.3.7) modal with long text inside the editor that causes a scroll. Set toolbarSticky: true.
2. Scroll down.
3. The toolbar is not visible.

When settings scrollableContainer to the modal, you can see that the toolbar "clips" when you scroll past it. However, it's not visible when scrolling further down.

[Fiddle](http://jsfiddle.net/lehti29/cj1d06su/)

##### Editor version.
3.2.3

##### OS.
Windows 10 Pro 19041.508

##### Browser.
Chrome Version 86.0.4240.75 (Official Build) (64-bit)

##### Recording.
![modal](https://user-images.githubusercontent.com/23076957/97961089-e9461300-1db2-11eb-9981-c2c60126cee2.gif)

**Comment by rizalrhmn** [comment]
yes i'm experiencing this too. Also when we change to another tab, and get back again, if the toolbar position is already passing the browser header, the sticky will work. But strangely, it won't back to the editor again (still floating). If we change tab again and the toolbar position is not passing the browser header, it will get back to the top of the editor (not floating).

the behaviour is very similar with this issues : https://github.com/froala/wysiwyg-editor/issues/2044

I'm using the latest Froala, and bootstrap version.

**Comment by lehti29** [comment]
Yeah I had some similar issues as well with the toolbar not getting the correct position after it was sticky and then lost focus.

Check out this [fiddle](http://jsfiddle.net/ilyasfroaal/9rp4ecx8/ ), which solved my "main" problem at least with the toolbar not being sticky. But had to tweak a bit more to get it working appropriately in IE11.

**Comment by rizalrhmn** [comment]
@lehti29 from what i'm seeing, you only add position:sticky to .modal-body and .fr-toolbar classes and add z-index to fr-toolbar ?

Unfortunately I can't make it work still :(. In my case, the toolbar is inside several other divs, not directly inside the .modal-body. And I'm using react-bootstrap & react-wysiwyg-froala. I don't know maybe that's affecting the result.

**Comment by lehti29** [comment]
Yeah correct!

Oh okay, how unfortunate! Yeah, might be differences in how we structure the HTML and different versions of bootstrap and so on, that might affect the result.

**Comment by AnastasiiaPavlenko** [comment]
Same problem here, neither toolbarSticky nor toolbarStickyOffset work. Did you manage to make it work in the end?

**Comment by ilyaskarim** [comment]
We’re closing this issue due to inactivity. If you’re still experiencing this problem on the latest version of our editor, please feel free to reopen the issue and let us know. You can find details on the latest updates here: [Froala Editor Changelog](https://froala.com/wysiwyg-editor/changelog/).
